bridge/github/config.go

```diff
@@ -205,13 +206,58 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("The remotes are:")
+
+		repos, nil := repo.ListRemotes()
+		reposMap := make(map[int]string)
+		i := 1
+		for _, value := range repos {
+
+			if checkGithubURL(value) {
+				fmt.Printf("[%d]: %s\n", i, value)
```

Comment:
Could you make sure to match the following UI (display a correct and short URL, space between the detected project and the custom option, `Detected projects`) ?

```
Detected projects:
[1]: github.com/MichaelMure/git-bug
[2]: github.com/adamslc/git-bug

[0]: another project
```